pages/autosuggest/virtual-resize.page.tsx

```diff
@@ -10,7 +10,8 @@ declare const window: ExtendedWindow;
 
 const options = [
   {
-    value: 'A very long option label that wraps upon resizing',
+    value:
+      'A very very very very very very very very very very very very very long option label that wraps upon resizing',
```

Comment:
Extended this string so that it still wraps when needed by the integration test.